### DISCUSSION TITLE
微调后checkpoint文件夹中没有pytorch.bin文件

### DISCUSSION THREAD

**Discussion by gsh206** [zai-org/ChatGLM3#971] [Q&A]
微调后checkpoint文件夹中没有pytorch.bin文件，过程中也没有报错呀
![1710319044796](https://github.com/THUDM/ChatGLM3/assets/51772411/1bfc18c3-a64e-4577-8595-0b584a80421c)

**Comment by gsh206** [comment]
求助求助求助

**Comment by gsh206** [comment]
transformers版本为4.38.1
看有别人说降低到4.30.2，但是降低了又报错就没继续

**Comment by carseny** [comment]
微调的模型参数是存在adapter_model.safetensors里的吧，推理的时候直接加载这个文件夹就行了

**Comment by ghost** [comment]
请问您的问题解决了吗？我也遇见了这样的问题，按照官方的lora_finetune.ipynb 文件一步步执行的，现在有点解决不了

**Comment by zRzRzRzRzRzRzR** [comment]
lora本来就没有啊，使用的ckpt是原始的，只有一个adapter
